FE/src/components/Admin.jsx

```diff
@@ -1,8 +1,301 @@
+                    setFail(false);
+                    setSuccess(true);
+                    // setUserBio(true);
+                    // setUserForm(false);
+                } else {
+                    setFail(true);
+                    setSuccess(false);
+                }
+
+            } catch (error) {
+                setError(error.message);
+                console.log( error );
+            }
+        }allUserCheck();
+    }, [])
+
+    async function signIn(){
+        navigate('/login');
+    }
+
+    useEffect(()=>{
+        async function allReviewedRecipesCheck () {
+            try {
+                // const response = await fetch(`${API}recipes/reviewedRecipes, {
+                //     method: "GET",
+                //     headers: {
+                //         "Content-Type":"application/json",
+                //         "Authorization": `Bearer ${token}`,
+                //     },
+                // });
+                // const result = await response.json()             
+                if (token !== null) {//need an elseif to crosscheck admin
+                    setReviewedRecipesList(dummyReviewedRecipes);
+                    setFail(false);
+                    setSuccess(true);
+                    // setUserBio(true);
+                    // setUserForm(false);
+                } else {
+                    setFail(true);
+                    setSuccess(false);
+                }
+
+            } catch (error) {
+                setError(error.message);
+                console.log( error );
+            }
+        }allReviewedRecipesCheck();
+    }, [])
+
+    useEffect(()=>{
+        async function allTagsCheck () {
+            try {
+                // const response = await fetch(`${API}tags, {
+                //     method: "GET",
+                //     headers: {
+                //         "Content-Type":"application/json",
+                //         "Authorization": `Bearer ${token}`,
+                //     },
+                // });
+                // const result = await response.json()             
+                if (token !== null) {//need an elseif to crosscheck admin
+                    setTagsList(dummyTags);
+                    setFail(false);
+                    setSuccess(true);
+                    // setUserBio(true);
+                    // setUserForm(false);
+                } else {
+                    setFail(true);
+                    setSuccess(false);
+                }
+
+            } catch (error) {
+                setError(error.message);
+                console.log( error );
+            }
+        }allTagsCheck();
+    }, [])
 
     return (
-        <><p>This is the Admin page for admin specific tasks</p></>
+        <><p>This is the Admin page for admin specific tasks</p>
+        <div className = 'wrapper'>
+            <div className='adminNav'>
+                <button onClick={() => {
+                    setAllUsers(true);
```

Comment:
We can look at this later, but this is probably better handled with react router rather than all these state changes.